recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -92,3 +92,10 @@ do_install:append() {
         sed -i -e "s/^xwayland=true/#xwayland=true/g" ${D}${sysconfdir}/xdg/weston/weston.ini
     fi
 }
+
+do_install:append:mx93-nxp-bsp() {
```

Comment:
Would it be possible to adjust the permissions so we could avoid this process to run as root?

Reply:
I would agree for `/dev/pxp_device` but I have no idea if it is a good idea to lift the accesspermissions for `/dev/dma_heap/linux,cma*`

Reply:
@thochstein Is it possible for you to check internally what are the devices that we need to provide access?